### PR TITLE
Tools#2 homework

### PR DESCRIPTION
**Pull request by malinaq3**

### CODE REVIEW

File: src/functions.cpp [hunk 1]
```diff
@@ -1,10 +1,9 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
-#include <algorithm>
+#include "functions.hpp"
 #include <iostream>
-#include <limits>
+#include <algorithm>
```
Comment (Marcinati) [inline src/functions.cpp]: Please sort headers.

File: src/functions.cpp [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (unsigned int i = 0; i < numbers.size(); ++i) {
```
Comment (Marcinati) [inline src/functions.cpp]: Read about std::size_t (instead of unsigned int).

File: src/functions.cpp (hunk 1 above)
Comment (malinaq3) [reply src/functions.cpp]: ok, thanks. I did it before watching last video

File: src/functions.cpp (hunk 2 above)
Comment (malinaq3) [reply src/functions.cpp]: got it and going to change it. Thanks again

**Review by DamiXioR — COMMENTED** [review]
good job

**Comment by Marcinati** [comment]
Now it's allright. :)

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**